illumos-utils/src/running_zone.rs

```diff
@@ -440,6 +440,9 @@ pub enum InstallZoneError {
 pub struct InstalledZone {
     log: Logger,
 
+    // Filesystem path of the zone
+    zonepath: PathBuf,
+
```

Comment:
Within this PR, this is always `/zone`, which resides in the ramdisk. However, this will change in the future, as some zones move to:

```
/pool/ext/<UUID>/zone
```